README.md

```diff
@@ -8,244 +8,22 @@ Build ([master](https://github.com/aws/aws-toolkit-vscode/tree/master) branch)|[
-            }
-        }
-    }
-    ```
-
-    Your `tasks.json` file should now look similar to this:
-
-    ```jsonc
-    {
-        "version": "2.0.0",
-        "tasks": [
-
-            <In here you may have 0 or more additional task entries>
-
-            {
-                "label": "launchLambdaFunction",
-                ... remainder of launchLambdaFunction entry would be here, omitted for brevity ...
-            }
-        ]
-    }
-    ```
-
-  * Save the `tasks.json` file
-
-* Define a Debug Configuration to attach to the task you just defined. From the **Command Palette**, select `Debug: Open launch.json`.
-
-  * If you are asked to Select an Environment, select `Node.js`, and the file will be created for you. You can delete the `Launch Program` configuration entry that was initially created in the file.
-
-* You should now see a `launch.json` file that looks similar to this:
-
-    ```javascript
-    {
-        "version": "0.2.0",
-        "configurations": [
-
-            <There may be 0 or more launch cofigurations in here already>
-
-        ]
-    }
-    ```
-
-    Add the following configuration entry into the `configurations` array
-
-    ```javascript
-    {
-        "type": "node",
-        "request": "launch",
-        "name": "Debug Local Lambda: <your function name here>",
-        "preLaunchTask": "launchLambdaFunction",
-        "address": "localhost",
-        "port": 5858,
-        "localRoot": "${workspaceFolder}/hello-world", // This is the workspace relative location of the folder referenced by your Serverless Application template resource's CodeUri
-        "remoteRoot": "/var/task",
-        "protocol": "inspector",
-        "skipFiles": [
-            "/var/runtime/node_modules/**/*.js",
-            "<node_internals>/**/*.js"
-        ]
-    }
-    ```
-
-    Your `launch.json` file should now look similar to this:
-
-    ```javascript
-    {
-        "version": "0.2.0",
-        "configurations": [
-
-            <There may be 0 or more launch cofigurations in here already>
-
-            {
-                "type": "node",
-                "request": "launch",
-                "name": "Debug Local Lambda: <your function name here>",
-                "preLaunchTask": "launchLambdaFunction",
-                .
-                .
-                .
-            }
-        ]
-    }
-    ```
-
-* Save the `launch.json` file
-
-You should now be able to switch to the Debug Pane in VS Code, and select your configuration from the dropdown menu. Pressing play or `F5` will launch and debug the lambda function you have just configured.
+* [Debugging NodeJS Lambda Functions](docs/debugging-nodejs-lambda-functions.md)
+* [Debugging Python Lambda Functions](docs/debugging-python-lambda-functions.md)
+<!-- TODO: Uncomment once the PR for this doc is merged -->
+<!-- * [Debugging .NET Core Lambda Functions](docs/debugging-dotnetcore-lambda-functions.md) -->
```

Comment:
Can we debug .NET apps through the debug panel with those instructions? If so, you can merge earlier if we explicitly state that it's not doable via codelens (and with a hefty dose of "EXPERIMENTAL"s scattered throughout)

Reply:
The doc in the link needs to exist before this is merged. Agree that we don't have to wait for .NET Core codelens support.